earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/common/utils.py

```diff
@@ -328,9 +328,12 @@ def RunCmd(os_cmd):
   try:
     if isinstance(os_cmd, str):
       os_cmd = shlex.split(os_cmd)
-    results = subprocess.check_output(os_cmd)
+    p = subprocess.Popen(os_cmd, shell=False,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.PIPE)
+    results = p.communicate()[1]
```

Comment:
Q: Do we not want to capture the standard error output, and log it, or show it to the user when the command fails?

Reply:
when the command fails, we would have captured  'results' and it will be returned to the calling clients as such. Does not the above comment and code changed address this (error handling) ?

Reply:
You're right. :stuck_out_tongue: The `results` variable seems like it would have the standard error of the process from the code above, but not the standard output.